keyboards/ut472/readme.md

```diff
@@ -0,0 +1,15 @@
+# ut47.2
+
+Somewhere between ortholinear and standard offset. Created to have all the same functions on a Planck in a keyboard but with a more conventional keyboard row stagger.
```

Comment:
Minor grammar correction:

```suggestion
Somewhere between ortholinear and standard offset. Created to have all the same functions on a Planck in a keyboard but with a more-conventional keyboard row stagger.
```

Reply:
This is not correct. You don't add a dash with comparatives

Reply:
no hyphen necessary.

Reply:
Respectfully, you are mistaken. The adjective "more-conventional" modifies the noun "keyboard row stagger." If this sentence had read "Created to have all the same functions on a Planck, but with a keyboard row stagger that is more conventional," I'd agree with you that the hyphen is not necessary.

- https://www.grammar-monster.com/lessons/hyphens_in_compound_adjectives.htm
- https://en.wikipedia.org/wiki/Compound_modifier
- https://www.quora.com/What-are-hyphenated-adjectives-What-are-some-examples
- https://getitwriteonline.com/articles/hyphenated-adjectives/

Reply:
I have never seen the phrase "more conventional", or indeed, "more *x*", written with a hyphen. Wikipedia claims that it's not normally hyphenated:
https://en.wikipedia.org/wiki/English_compound#Hyphenated_compound_adjectives

And IMO the indefinite article removes any ambiguity here.

Reply:
Perhaps this is an area of grammar which lacks consensus, or changed over time. All through school I was taught that one should hyphenate when using *more* or *most* if the compound is a singular descriptor:

- "a more-likely scenario" or "the more-likely scenario"
- "a most-respected member" or "the most-respected member"

... because the compound modifies the noun. :man_shrugging:

Reply:
Sounds like someone in your school system might have overgeneralised a rule. English is defined more by it's exceptions, after all :) My personal experience is that I never see *more* or *most* used with a hyphen, anywhere.